lj/lj.py

```diff
@@ -49,17 +52,21 @@ class LJServer:
     host: server to connect to.  Defaults to the official LiveJournal server.  Note that
         it is assumed everything on this server is in the same location as it is on
         livejournal.com.
+    ssl: Transport/SafeTransport for http/s - needed if using this lib for dreamwidth.org
```

Comment:
Please remove reference to this project — SSL support can make sense for any project, not just one you mention